packages/@react-spectrum/table/src/TableView_DEPRECATED.tsx

```diff
@@ -286,7 +295,9 @@ function TableView_DEPRECATED<T extends object>(props: SpectrumTableProps<T>, re
             {
               'spectrum-Table--quiet': isQuiet,
               'spectrum-Table--wrap': props.overflowMode === 'wrap',
-              'spectrum-Table--loadingMore': state.collection.body.props.loadingState === 'loadingMore'
+              'spectrum-Table--loadingMore': state.collection.body.props.loadingState === 'loadingMore',
+              'spectrum-Table--isScrollingVertically': isVerticalScrollbarVisible,
```

Comment:
Hmm, for some reason https://reactspectrum.blob.core.windows.net/reactspectrum/4c2cc4b1e72a9ef6d276743560991dcd9110b47d/storybook/index.html?path=/story/tableview--selectionstyle-highlight doesn't get this scrollinVertically class applied when I have scrollbars on always:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/9661127/167194653-8504c9cb-a98a-4c69-9020-145d889a8b96.png">

Works just fine if I have scrollbars on only when scrolling